ui/src/organizations/components/CreateOrgOverlay.tsx

```diff
@@ -108,4 +117,17 @@ export default class CreateOrgOverlay extends PureComponent<Props, State> {
       errorMessage: '',
     })
   }
+
+  private randomErrorMessage = (key: string): string => {
```

Comment:
```suggestion
  private randomErrorMessage(key: string): string {
```

Reply:
I suppose this function doesn't need the binding that comes from using an arrow function